### PR TITLE
Feat: Add support for custom tailwind config file path

### PR DESCRIPTION
**Pull request by NicolasLopes7**
This PR adds support for customizing the Tailwind config file path through command-line arguments. 
The change has been implemented for the CLI entry point and further work is needed to pass the parameters to the Babel plugin. Would love to get your eyes and ship the Babel plugin support with that PR! Cheers. Thank you! 🎉

### CODE REVIEW

File: examples/vite-example/package.json [hunk 1]
```diff
@@ -18,7 +18,6 @@
     "@types/react": "^18.0.26",
     "@types/react-dom": "^18.0.9",
     "@vitejs/plugin-react": "^3.0.0",
-    "typewind": "*",
```
Comment (NicolasLopes7) [inline examples/vite-example/package.json:21]: this dependency key is dup, so I removed it

**Comment by Mokshit06** [comment]
thankyou! I think rather than having it as a cli arg, this customPath should be declared in `package.json` under typewind config or in a custom config file like :-

```json
{
    "typewind": {
        "configPath": "./tailwind.config.js`
    }
}
```

That way both the Babel plugin and type generator can easily access this property.

Keeping it as a cli arg would require passing the custom path in 2 places which could lead to them not being in sync

**Comment by NicolasLopes7** [comment]
> 



> thankyou! I think rather than having it as a cli arg, this customPath should be declared in `package.json` under typewind config or in a custom config file like :-
> 
> ```json
> {
>     "typewind": {
>         "configPath": "./tailwind.config.js`
>     }
> }
> ```
> 
> That way both the Babel plugin and type generator can easily access this property.
> 
> Keeping it as a cli arg would require passing the custom path in 2 places which could lead to them not being in sync

Love that! Will do!

**Comment by NicolasLopes7** [comment]
Just pushed the changes @Mokshit06 ! Let me know if you have any concern about that

**Comment by kavinvalli** [comment]
I think the docs for this doesn't really fall under the `Usage` heading. Should have this separately or under `Installation`. What do you say @Mokshit06?

File: examples/vite-example/package.json (hunk 1 above)
Comment (kavinvalli) [reply examples/vite-example/package.json:21]: Thanks for noticing!

**Comment by NicolasLopes7** [comment]
> I think the docs for this doesn't really fall under the `Usage` heading. Should have this separately or under `Installation`. What do you say @Mokshit06?

I think it makes sense, gonna change it!

**Comment by Mokshit06** [comment]
great work, thanks! just a suggestion: i would prefer the version to be left as is so that I can update it in a release hook or script, but since typewind doesn't have one right now i'll leave it as is.

**Pull request merged**